tools/api-builder/angular.io-package/templates/api-list-data.template.html

```diff
@@ -7,7 +7,7 @@
       "docType": "{$ item.docType $}",
       "stability": "{$ item.stability $}",
```

Comment:
both stability and security fields can contain free-form text. we should escape those too.